hook!-full-game/src/UI/debug/DebugPanel.gd

```diff
@@ -0,0 +1,38 @@
+extends Control
+
+onready var reference_node: = get_node(reference_node_path)
+onready var container: VBoxContainer = $MarginContainer/VBoxContainer
+
+export (NodePath) var reference_node_path: NodePath = "."
+export (PoolStringArray) var properties = []
+
+func _ready() -> void:
+	setup_properties()
+
+
+func _process(delta) -> void:
+	for property_string in properties:
+		update_property(property_string, reference_node)
+
+
+func setup_properties() -> void:
```

Comment:
All methods are public. Am I supposed to use this one? How about simpler names like clear() and reset()?

Reply:
No, the update method should be private, `_clear` as well, add_property_label and setup are public. You can see in the example I just pushed